woocommerce/woocommerce
# Introduce missing button element classes for the account orders and downloads

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Before starting ensure that you are using a block theme like TT2, or TT3.

**Testing in order-again context:**

1. Create a new order and get to the Order Received page (don't close it)
2. Process the order from wp-admin and refresh the previous page to see the "Order again" button.
3. Notice that it's missing some styles. See https://d.pr/i/oVXIas
4. Apply this fix, and ensure the button appears correctly.

**Testing in the pagination buttons on the account page:**
1. Navigate to My Account > Orders page and ensure that you have enough so the pagination shows up _(hint: one could use the `woocommerce_my_account_my_orders_query` filter to wire a small number (e.g., 2) in the `limit` key for the query)_
2. Notice the buttons are missing styles. See https://d.pr/i/lYs1jg
3. Apply this fix, and ensure the buttons appear correctly.

**Testing in the no-downloads cta button on the account page:**
1. Navigate to My Account > Downloads page and ensure your user hasn't downloaded yet.
2. Check the CTA button inside the notice saying "No downloads available yet"; it shouldn't look right.
3. Apply this patch and ensure the CTA shows up correctly.
